example/todos/src/app/components/count/count.component.ts

```diff
@@ -0,0 +1,15 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { Todos } from '../../stores/todos.store';
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  selector: 'app-count',
+  template: `
+    <span id="todo-count" *mobxAutorun><strong>{{todos.uncompletedItems}}</strong>
```

Comment:
Personally, I'd put the `<strong>...</strong>` on its own line.

Reply:
done!